packages/devtools_app/lib/src/memory/flutter/memory_screen.dart

```diff
@@ -78,6 +78,69 @@ class MemoryBodyState extends State<MemoryBody> {
     );
   }
 
+  static const String _liveFeed = 'Live Feed';
+  String memorySource = _liveFeed;
+
+  Widget createMenuItem(String name) {
+    final rowChildren = memorySource == name
+        ? [
+            Icon(Icons.check, size: 12),
+            const SizedBox(width: 10),
+            Text(name),
+          ]
+        : [
+            const SizedBox(width: 22),
+            Text(name),
+          ];
```

Comment:
you could use list comprehensions to simplify defining the row children.
For example:
```dart
[
  if (memorySource == name)
    Icon(Icons.check, size: 12),
  const SizedBox(memorySource == name ? 10 : 22),
  Text(name),
]
```

Reply:
When I changed to use a dropdown I eliminated this code.